source/fundamentals/connection.txt

```diff
@@ -55,10 +55,15 @@ options. In the example, we set two connection options:
 ``maxPoolSize=20`` and ``w=majority``. To learn more about connection
 options, read the :ref:`golang-connection-options` section of this guide.
 
-The following code shows how you can use a connection string in a
-client to connect to MongoDB. In this example, the client uses 
-the :ref:`Stable API <golang-stable-api>` feature, though
-you can connect to MongoDB without this feature.
+To connect to MongoDB, create a client by passing your connection string
```

Comment:
Suggestion:
On a second read, I think the term "client" needs an introduction and this seems to be a good place to transition. I think one of the better solutions is to separate the explanation of how the Connection URI relates to the client into a sub-section since this one is about the Connection URI.

E.g.


```
MongoDB Client
~~~~~~~~~~~~~~

You need to create a MongoDB client to manage your connection 
and run database commands. To create a client, call the ... method 
and pass it your connection string and any options...

< perhaps link to some docs about the client and options >

In the options, you can set the Stable API version. The Stable API 
option ensures that your application runs commands that are
compatible with future server versions... Learn more about...
```